packages/workspace/src/browser/workspace-service.ts

```diff
@@ -261,10 +261,18 @@ export class WorkspaceService implements FrontendApplicationContribution {
     }
 
     /**
-     * Returns `true` if there is an opened workspace in theia, and the workspace has more than one root.
+     * Returns `true` if there is an opened workspace, and theia supports having more than one root.
```

Comment:
I'm not sure if we should add `... and theia supports having more than one root.` as we don't check the preference value here. Something like below would be more accurate, what do you think?

```ts
// Returns `true` if a multiple-root workspace is currently open.
```

Reply:
you are right. i will change it.

Reply:
you are right. i will change it.